README.md

```diff
@@ -33,6 +32,45 @@ There's a `.env.sample-dev` file provided which you can rename to `.env` if you
 
 You can chat to us in [Keybase](https://keybase.io). Download their chat app, then click **Teams -> Join a team**. Type **nymtech.friends** into the team name and hit **continue**. For general chat, hang out in the **#general** channel. Our development takes places in the **#dev** channel. Node operators should be in the **#node-operators** channel.
 
+### Rewards
+
+Node, node operator and delegator rewards are determined according to the principles laid out in the [Nym Tokenomics paper](needs-link). Below is a TLDR of the variables and formulas involved in calculating the epoch rewards. Initial reward pool is set to 250 million Nym, making the circulating supply 750 million Nym.
```

Comment:
I don't think we are going to be releasing the tokenomics paper anytime soon (?). But instead I think you can just say that it's laid out in section 6.5 of our whitepaper and link to that instead (https://nymtech.net/nym-whitepaper.pdf)

Reply:
Or maybe just "section 6"